tests/test_threading.py

```diff
@@ -168,6 +175,7 @@ def test_model_context(self):
 
 
 # Same as TestLockedTransition but with LockedHierarchicalMachine
+@skipIf(pgv is None, 'AGraph diagram requires pygraphviz')
```

Comment:
I' not sure how can I locate that line?

A nice point about `unittest.skipIf` is that you get the report of skipped TCs at the end.